app/_redirects

```diff
@@ -42,6 +42,31 @@
 /konnect/getting-started/configure-runtime/ /konnect/getting-started/configure-data-plane-node/
 /konnect/api/runtime-groups-config/overview/ /konnect/api/
 
+# API deprecation
```

Comment:
```suggestion
# API markdown doc deprecation
```
just in case anyone is looking at this file later and gets worried about a deprecated API.